src/HTCondor/condor-9-to-10-upgrade-check.py

```diff
@@ -404,43 +434,62 @@ def check_gpu_requirements():
         if len(ppl_using_gpu) > 0:
             format_print(
                 f"There are currently {len(ppl_using_gpu)} user(s) using V9 GPU requirements specification.",
-                8,
+                offset=8,
             )
             format_print(
                 "The method to request GPUs changes in V10 to use the new submit description keywords:",
-                8,
+                offset=8,
             )
-            format_print("- request_gpus", 16)
-            format_print("- require_gpus", 16)
+            format_print("- request_gpus", offset=16)
+            format_print("- require_gpus", offset=16)
             format_print(
                 "The following user(s) were found during the query for the old style of requesting GPUs:",
-                8,
+                offset=8,
                 newline=True,
             )
             for user in ppl_using_gpu:
-                format_print(user, 16)
+                format_print(user, offset=16)
     except htcondor.HTCondorLocateError:
-        format_print("Failed to locate local schedd.", 8, err=True)
+        format_print("Failed to locate local schedd.", offset=8, err=True)
+
+
+def usage():
+    """Print Script Usage"""
+    format_print("HTCondor Upgrade Incompatabilities Script", newline=True)
+    format_print(
+        """  This Script is to help assist catching possible issues that
+  may occur while upgrading an HTCondor system from
+  version 9 to version 10.""")
+    format_print("Options:", newline=True, offset=8)
+    format_print("-h/-help         Display Usage", offset=12)
+    format_print("-ce              This host is an HTCondor-CE", offset=12)
+    format_print("-ignore-gpu      Skip checking GPU requirements incompatability", offset=12)
+    sys.exit(0)
 
 
 def main():
-    # Make sure script is running as root because we need to run 'condor_token_list'
-    if os.geteuid() != 0:
-        format_print("Error: Script not ran as root")
-        sys.exit(1)
     # Systems with config to check: Base condor and condor-ce
     ecosystems = ["HTCondor"]
+    check_gpu_reqs = True
     # Check if passed -ce flag to indicate this host is a CE
     for arg in sys.argv:
```

Comment:
Ah, I forgot to mention that we should probably be using the `argparse` library here, but it's probably a bit too late for that. Anyway, the first arg is always the program name so can probably skip that:

```suggestion
    for arg in sys.argv[1:]:
```